content/ros/ROS_Nodes_Topics_Messages/home_work/occupancy_grid_maps.md

```diff
@@ -0,0 +1,21 @@
+# Occupancy Grid Maps
+
+- Occupancy grid maps are a popular map representation used in robotics. 
+
+![example](figures/map.png)
+
+- An occupancy grid consists of cells, each cell represents an area in the environment. A cell can be occupied, free, or unknown.
+
+ - An occupancy grid can be visualized as a grayscale image. In memory they are stored as an array of occupancy values. Each element in the array corresponds to a cell which represents certain area of the environment. 
+
+- The example below shows a simple occupancy grid (a very coarse map just for demonstration):
+
+
+On the left is the real environment, on the right end is the occupancy grid map that models/represents the environment.
+![grid](figures/occupancy_grid.png) 
+
+In memory it could be stored as:
+```python
+ map data = [0, 0, 0, 0, ... , 0, 100 ,100 ,100 ,0 ,0 , ... ]
+ ```
+- Note that this array is not useful unless more information (meta information) is given about the map. Such as the cell size (map resolution), origin of map, size of map, etc..
```

Comment:
Put the `such as ...` part in parentheses rather than as a new sentence.

Reply:
done